### PR TITLE
Update item.md

### PR DESCRIPTION
**Pull request by Mohron**
Might want to check my en_GB spelling :D

### CODE REVIEW

File: docs/modules/item.md [hunk 1]
```diff
@@ -35,7 +35,20 @@ and it would involve removing a current enchantment, add the flag `-o`. If you a
 enchantment level onto an item which is not normally allowed (often referred to as "unsafe"), add the flag `-u`, though
 this requires an extra permission (`nucleus.enchant.unsafe`).
 
+## Repair
+
+Nucleus can be used to repair items in a player's inventory using `/repair`. By default, `/repair` will only target the item held in a player's main hand. By using and combining the availible flag arguments, specific inventory locations or the entire inventory can be repaired.
```
Comment (dualspiral) [inline docs/modules/item.md]: I didn't realise the `en_US` spelling was "availible" ;)

(it's "available")

File: docs/modules/item.md [hunk 2]
```diff
@@ -35,7 +35,20 @@ and it would involve removing a current enchantment, add the flag `-o`. If you a
...
+
+### Flag Arguments
+
+* `/repair -m` repairs the player's main hand (default behavior).
```
Comment (dualspiral) [inline docs/modules/item.md]: If we're going all `en_GB` (and let's face it, we all know that's the _correct_ way to spell!), you're missing a "u" in "behaviour" ;)

In all honesty, I'd not thought about the UK/US differences when writing these so I can't say I'm horribly bothered, but I guess we should be consistent.

File: docs/modules/item.md (hunk 1 above)
Comment (Mohron) [reply docs/modules/item.md]: I never claimed to be good at spelling.

File: docs/modules/item.md (hunk 2 above)
Comment (Mohron) [reply docs/modules/item.md]: I personally was going for consistency.

**Comment by dualspiral** [comment]
Ta muchly.

**Comment by Mohron** [comment]
Somebody's gotta document the new features, might as well be the jerk who added them!

**Comment by dualspiral** [comment]
Don't say that! That puts a lot on my plate! :P

**Pull request merged**